db/migrate/20200806112501_expand_cloud_volume.rb

```diff
@@ -0,0 +1,8 @@
+class ExpandCloudVolume < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :cloud_volumes, :storage_resource, :type => :bigint, :index => true
+    add_reference :cloud_volumes, :storage_service, :type => :bigint, :index => true
+    add_column :cloud_volumes, :compliant, :boolean
+    add_column :cloud_volumes, :volume_source, :string
```

Comment:
What does this column mean?

Reply:
As we are re-using the cloud volume to represent the volumes managed by autosde, this column will indicate what is the origin of this volume, mainly to be easily displayed as info in the UI list of volumes.

Reply:
What would some of the possible values be?  E.g. does this indicate that it came from autosde (which we could get from the ext_management_system link) or something else?